framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -1068,4 +1069,25 @@ public static boolean matches(String actual, String expected) {
       }
     return true;
   }
+
+  public static String getFrameworkVersion() {
+    String commit = "";
+    String commitAuthor = "";
+    String commitMail = "";
+    String commitMessage = "";
+    try {
+      URL u = Resources.getResource("framework.git.properties");
+      if (u != null) {
+        Properties p = new Properties();
+        p.load(Resources.asByteSource(u).openStream());
+        commit = p.getProperty("git.commit.id");
+        commitAuthor = p.getProperty("git.commit.user.name");
+        commitMail = p.getProperty("git.commit.user.email");
+        commitMessage = p.getProperty("git.commit.message.short");
+      }
+    } catch (IOException | IllegalArgumentException e) {
+      LOG.warn("Failure while trying to load \"framework.git.properties\" file.", e);
```

Comment:
In case of Exception or `framework.git.properties` not available, the **DRILL TEST FRAMEWORK VERSION** would look something like
```
Commit: null
Author: null <null>


null
```